src/netaddress.h

```diff
@@ -244,6 +244,11 @@ class CNetAddr
             }
         }
 
+        template <typename Stream, typename Operation>
+        inline void SerializationOp(Stream& s, Operation ser_action) {
+            READWRITE(m_addr);
+        }
+
```

Comment:
What is this used for? I don't see it in the master Bitcoin branch.
Everything else looks good to me :+1:

Reply:
Oops yeah this is extraneous, don't know why it was pulled in when cherry-picking. Going to remove that and also revert the changes to the serialization and stream tests to make this as unobtrusive as possible.